ccd-definition/ComplexTypes/BusinessProcess.json

```diff
@@ -0,0 +1,24 @@
+[
+  {
+    "ID": "BusinessProcess",
+    "ListElementCode": "processInstanceId",
+    "FieldType": "Text",
+    "ElementLabel": "Camunda process instance Id",
+    "SecurityClassification": "Public"
+  },
+  {
+    "ID": "BusinessProcess",
+    "ListElementCode": "status",
+    "FieldType": "FixedList",
+    "FieldTypeParameter": "BusinessProcessStatus",
+    "ElementLabel": "Camunda business process status",
+    "SecurityClassification": "Public"
+  },
+  {
+    "ID": "BusinessProcess",
+    "ListElementCode": "activityId",
```

Comment:
I think it should be `taskId`

Reply:
nope, definitely activityId. When I get a task from Camunda I get these values included:

`
"activityId": "Activity_13h4s1i",
"id": "7ec7bb40-ff1b-11ea-9a89-0242ac130004",
`

The Id of the task is unique to each task. The activityId we can set from the Camunda Modeller (it's id at the top of the right menu). We need something common that is unique to each task but not for each instance of the task

Reply:
cool, thanks for clarifying 👍